DiceKitTests/Die_Tests.swift

```diff
@@ -404,4 +404,38 @@ extension Die_Tests {
         expect(result.value) == stubbedRoll
     }
     
+    func test_probabilityMass_shouldReturnCorrect() {
+        property("probability mass") <- forAll {
+            (sides: Int) in
+            
+            let inverseSides = -sides
+            
+            let die = Die(sides: sides)
+            let inverseDie = Die(sides: inverseSides)
+            
+            let probMass = die.probabilityMass
+            let inverseProbMass = inverseDie.probabilityMass
+            
+            guard sides != 0 else {
+                return probMass == ProbabilityMass.zero
+            }
+            
+            let outcomePerValue = 1.0/ProbabilityMass.Probability(abs(sides))
```

Comment:
Seems like following Arrange Act Assert this line would be in the middle by itself, let statements above it, and the remaining statements with returns below. Your call.